src/mod_ping.erl

```diff
@@ -56,12 +56,12 @@
          user_keep_alive/2,
          user_ping_response/4]).
 
--record(state, {host = <<"">>,
+-record(state, {host = jid:lserver(),
```

Comment:
Here the default value of the field is defined. `jid:lserver()` is a type so creation of the record will cause a undef exception when `host` is not specified.